edit-post/store/effects.js

```diff
@@ -108,11 +108,21 @@ const effects = {
 		// Collapse sidebar when viewport shrinks.
 		subscribe( onChangeListener(
 			() => select( 'core/viewport' ).isViewportMatch( '< medium' ),
-			( isSmall ) => {
-				if ( isSmall ) {
-					store.dispatch( closeGeneralSidebar() );
-				}
-			}
+			( () => {
```

Comment:
I feel this could be written more succinctly. Is the IIFE required?

Reply:
The `if ( sidebarToReOpenOnExpand ) {` could be removed, as it was before, but it is useful as it avoids dispatching an action to close the sidebar if it is already closed. The other conditions look necessary, unfortunately, I don't think we can remove them.